### PR TITLE
Implement stakers UI support

### PR DESCRIPTION
**Pull request by pablomendezroyo**
Implement stakers UI support

### CODE REVIEW

File: validator/entrypoint.sh [hunk 1]
```diff
@@ -54,7 +23,7 @@ exec -c lighthouse \
     --init-slashing-protection \
     --datadir /root/.lighthouse \
     --beacon-nodes $BEACON_NODE_ADDR \
-    --graffiti="${GRAFFITI:0:32}" \
+    --graffiti="$GRAFFITI" \
```
Comment (MysticRyuujin) [inline validator/entrypoint.sh:26]: Should put this limit back in place

File: GETTING_STARTED.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
-## Welcome to the Merge Ready Consensus Client: Lighthouse
+## Welcome to the Consensus Client: Lighthouse
 
-- Your keystores should now managed in the [Web3Signer-UI](http://ui.web3signer.dappnode?signer_url=http://web3signer.web3signer.dappnode:9000) 
-- Don't have the Web3signer installed yet? You may not have migrated to the Mainnet-Multiclient, which is necessary for the Merge. It can be manually installed [here](http://my.dappnode/#/installer/web3signer.dnp.dappnode.eth)
-- With this update the Execution Layer Client must be selected in the Consensus Client, and the chosen Consensus Client is selected in the [Web3Signer](http://my.dappnode/#/packages/web3signer.dnp.dappnode.eth/config)
+- Your keystores should now be managed in the [Web3Signer-UI](http://ui.web3signer.dappnode?signer_url=http://web3signer.web3signer.dappnode:9000)
+- Don't have the Web3Signer installed yet? It can be manually installed [here](http://my.dappnode/#/installer/web3signer.dnp.dappnode.eth)
+- All Staking Management has a new home in the Comprehensive [StakersUI](http://my.dappnode/#/stakers)
```
Comment (alexpeterson91) [inline GETTING_STARTED.md]:
```suggestion
- All Staking Management has a new home in the Comprehensive [StakersUI](http://my.dappnode/#/stakers/mainnet)
```
Is this gonna be /mainnet or /ETH to complete the URL @pablomendezroyo

File: beacon-chain/entrypoint.sh [hunk 3]
```diff
@@ -3,6 +3,37 @@
...
+"geth.dnp.dappnode.eth")
+    HTTP_ENGINE="http://geth.dappnode:8551"
+    ;;
+"nethermind.public.dappnode.eth")
+    HTTP_ENGINE="http://nethermind.dappnode:8551"
+    ;;
+"erigon.dnp.dappnode.eth")
+    HTTP_ENGINE="http://erigon.dappnode:8551"
+    ;;
+"besu.dnp.dappnode.eth")
```
Comment (alexpeterson91) [inline beacon-chain/entrypoint.sh]:
```suggestion
"besu.public.dappnode.eth")
```
Nabsku ended up publishing the public version which I think is missing s couple important commits at least for aesthetic purposes etc. For mainnet we have the goerli under our control published as s DNP.

File: beacon-chain/entrypoint.sh [hunk 4]
```diff
@@ -3,6 +3,37 @@
 # Concatenate EXTRA_OPTS string
 [[ -n "$CHECKPOINT_SYNC_URL" ]] && EXTRA_OPTS="${EXTRA_OPTS} --checkpoint-sync-url=${CHECKPOINT_SYNC_URL}"
 
+case $_DAPPNODE_GLOBAL_EXECUTION_CLIENT_MAINNET in
+"geth.dnp.dappnode.eth")
+    HTTP_ENGINE="http://geth.dappnode:8551"
+    ;;
+"nethermind.public.dappnode.eth")
+    HTTP_ENGINE="http://nethermind.dappnode:8551"
```
Comment (alexpeterson91) [inline beacon-chain/entrypoint.sh]:
```suggestion
    HTTP_ENGINE="http://nethermind.public.dappnode:8551"
```
Adopt for mainnet package.

File: beacon-chain/entrypoint.sh [hunk 5]
```diff
@@ -3,6 +3,37 @@
...
+    ;;
+"erigon.dnp.dappnode.eth")
+    HTTP_ENGINE="http://erigon.dappnode:8551"
+    ;;
+"besu.dnp.dappnode.eth")
+    HTTP_ENGINE="http://besu.dappnode:8551"
```
Comment (alexpeterson91) [inline beacon-chain/entrypoint.sh]:
```suggestion
    HTTP_ENGINE="http://besu.public.dappnode:8551"
```
Public package is published

**Review by alexpeterson91 — CHANGES_REQUESTED** [review]
Check my changes. and question on stakersUI link thanks

**Pull request merged**